.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -0,0 +1,44 @@
+name: "Feature request"
+description: "Request new goodies"
+title: "[Feature]: "
+labels: ["enhancement"]
```

Comment:
Similar comment to the one above: probably just setting the label is sufficient and we don't need to set the prefix for the title.

Reply:
removed